package.json

```diff
@@ -7,7 +7,8 @@
     "dev": "ts-node-dev --respawn --pretty --transpile-only src/server.ts",
     "testLocal": "dotenv -e .env.test npm run test",
     "test": "mocha --file ./test/helper.ts --require ts-node/register test/**/*-test.ts",
-    "start": "tsc && node dist/server.js"
```

Comment:
This start script ought not be removed, it is used in deployment. I assume this was just a merge conflict error.